mailpoet/lib/Subscribers/ImportExport/ImportExportRepository.php

```diff
@@ -170,6 +170,7 @@ public function updateMultiple(
       " . implode(' AND ', $keyColumnsConditions) . "
     ", $parameters, $parameterTypes);
     $this->notifyUpdates($className, $columns, $data);
+    $this->entityManager->clear($className);
```

Comment:
@rodrigoprimo `EntityManage::clear with entity name is deprecated. The issue is pretty much related to the investigation and changes @JanJakes did in https://github.com/mailpoet/mailpoet/pull/5315. We should use [the helper](https://github.com/mailpoet/mailpoet/pull/5315/commits/8457976c5f5c0933a255bb5a60db037e39ede3bc) introduced in Jan's PR.

Reply:
Thanks for the review. It makes sense to me to use the helper introduced by JJ in #5315. I guess we will have to wait for his PR to be merged. I will ask @websupporter how to proceed here since I will move to another team next week.

Reply:
@costasovo There is a complication for using the helper. The `ImportExportRepository` class doesn't inherit from `Repository` because it works with multiple entities and can't implement the `getEntityClassName` abstract method needed by the helper (confirmed by @lysyjan who made this class). 

What do you think would be the best workaround? Implementing a private version of the helper for this class which would work (we know the `className` and other constraints)?

Reply:
@alex-mpoet Hmm, I see that the method is called for SubscriberCustomFieldEntity and SubscriberEntity, so I guess we could also inject repositories for these two entities and call `detachAll` on those. In my opinion, better than duplicating the code somewhere else. Would such a solution work?

Reply:
Yes, that would work, thanks! I made the change.